components/brave_wallet_ui/components/desktop/portfolio-transaction-item/index.tsx

```diff
@@ -486,12 +482,12 @@ export const PortfolioTransactionItem = React.forwardRef<HTMLDivElement, Props>(
                 ? getLocale('braveWalletTransactionApproveUnlimited')
                 : normalizedTransferredValue}{' '}
               <AddressOrAsset
-                onClick={onAssetClick(txSymbol, txToken?.contractAddress)}
+                onClick={() => onAssetClick(txSymbol, txToken?.contractAddress)}
```

Comment:
```suggestion
                onClick={onAssetClick(txSymbol, txToken?.contractAddress)}
```

Reply:
Fixed binding to these callbacks. Completely missed that these are actually callback factories
https://github.com/brave/brave-core/pull/19586/commits/de8a47305ae846acfbfed7247e3d7b5a88096ad7